lib/participant.js

```diff
@@ -479,6 +479,7 @@ class Participant extends EventEmitter {
  * A {@link Track} was added by the {@link Participant}.
  * @param {Track} track - The {@link Track} that was added
  * @event Participant#trackAdded
+ * @deprecated
```

Comment:
For RemoteParticipants, we can suggest "trackSubscribed". For LocalParticipants, I don't believe we'll be emitting an event anymore?

Reply:
@markandrus In LocalParticipant, "trackAdded" will be used internally.